### PR TITLE
Add indentation to StyledSyntaxDocument

### PR DESCRIPTION
**Pull request by RonSkons**

### CODE REVIEW

File: src/test/resources/com/itextpdf/rups/view/itext/contentStreamSnippets/nestedIndented.cmp
Comment (Minothor) [inline src/test/resources/com/itextpdf/rups/view/itext/contentStreamSnippets/nestedIndented.cmp:1]: I like this one - I found myself double-checking myself how lines 10->11 should indent.
Good test!

File: src/main/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocument.java [hunk 1]
```diff
@@ -112,9 +118,9 @@ public boolean isMatchingOperands() {
      * @return {@code false} if binary stream content, {@code true} otherwise
```
Comment (Minothor) [inline src/main/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocument.java:118]: maybe change this to `... if binary stream content or indented, {@code ...` to account for the new constaint?

**Review by Minothor — COMMENTED** [review]
Code looks good, I'd suggest editing the one comment to update the documentation to match behaviour, but it's a minor thing.

File: src/main/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocument.java (hunk 1 above)
Comment (RonSkons) [reply src/main/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocument.java:118]: Indents are textual (or at least, not binary) so the function returns true in this case, as described by the comment.

Comment (Minothor) [reply src/main/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocument.java:118]: Ah fair point, I've delved deeper and while the Element.getAttributes() call can be used for generic blocks, you're right in that it should only be called for text elements here.

**Comment by Minothor** [comment]
Hey @RonSkons could you move your feature into a named branch please?
i.e. `improvement/StyledSyntaxDocument_Indentation` or the like?
Jenkins isn't happy with the changes being made on your fork's develop branch.

If you could squash your commits into one with a summary of hte changes too, that'll help us keep the commit log for the core repo cleaner.
All in all though, happy with this - cheers!

**Comment by RonSkons** [comment]
Done!

**Pull request closed without merging**